Resources/doc/filters.md

```diff
@@ -0,0 +1,98 @@
+# Filters
+
+The bundle provides a generic system to apply filters on collections. It ships with built-in Doctrine ORM support
+and can be extended to fit your specific needs.
+
+By default, all filters are disabled. They must be enabled manually.
+
+When a filter is enabled, it is automatically documented as a `hydra:search` property in collection returns. It also automatically
+appears in the NelmioApiDoc documentation if this bundle is installed and enabled.
+
+## Adding Doctrine ORM filters
```

Comment:
The "where" term feels more familiar too me but it may be due to my experience with StrongLoop. "search" is already used in Hydra and some mainstream frameworks like [Django](http://www.django-rest-framework.org/api-guide/filtering/#searchfilter) so +1 for it.